app/routes/circulars._archive._index/route.tsx

```diff
@@ -261,20 +261,39 @@ export default function () {
             Search
           </Label>
           <input type="hidden" name="view" value={view} />
-          <TextInput
-            autoFocus
-            className="minw-15"
-            id="query"
-            name="query"
-            type="search"
-            defaultValue={inputQuery}
-            placeholder="Search"
-            aria-describedby="searchHint"
-            onChange={({ target: { form, value } }) => {
-              setInputQuery(value)
-              if (!value) submit(form, { preventScrollReset: true })
-            }}
-          />
+          {isGroupView && (
+            <TextInput
+              autoFocus
+              className="minw-15"
+              id="query"
+              name="query"
+              type="search"
+              placeholder="Search eventID"
+              defaultValue={inputQuery}
+              aria-describedby="searchHint"
+              onChange={({ target: { form, value } }) => {
+                setInputQuery(value)
+                if (!value) submit(form, { preventScrollReset: true })
+              }}
+            />
+          )}
+          {!isGroupView && (
+            <TextInput
+              autoFocus
+              className="minw-15"
+              id="query"
+              name="query"
+              type="search"
+              placeholder="Search"
+              defaultValue={inputQuery}
+              aria-describedby="searchHint"
+              onChange={({ target: { form, value } }) => {
+                setInputQuery(value)
+                if (!value) submit(form, { preventScrollReset: true })
+              }}
+            />
+          )}
+
```

Comment:
so instead of having mostly the same thing here twice, if you declare a variable that holds the search text based on view, you can do something like this:
`const searchText = isGroupView ? 'Event Name' : 'Search'`
and then where it has 
`placeholder={searchText}`